### PR TITLE
feat(get-age-string): wrote test for util

### PR DESCRIPTION
**Pull request by Vladosik12333**
dev
## GitHub Project

* [Task](https://localhost-vlad.atlassian.net/jira/software/projects/EEF/boards/3?selectedIssue=EEF-64)

## Code reviewers

- [ ] @andrii-kuzmichov | @VadymKharyton 

### Second Level Review

- [ ] All

## Summary of issue

Test for util.

## Summary of change

Create file get-age-string.test.js

## Testing approach

Yes, in console (npm run test).

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +30,7 @@
 		"material-ui": "0.20.2",
 		"material-ui-flat-pagination": "3.2.0",
 		"material-ui-pagination": "1.1.6",
-		"mdbreact": "4.19.0",
+		"mdbreact": "^5.2.0",
```
Comment (andrii-kuzmichov) [inline package.json]: why do we need it?

File: src/components/helpers/get-age-string.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    expect(result).toBe("---");
+  });
+  it("test with less 1 years", () => {
+    const result = getAge("10/10/2022");
+    expect(result).toBe(0);
+  });
+  it("test without prop", () => {
+    const result = getAge();
+    expect(result).toBe(0);
+  });
```
Comment (andrii-kuzmichov) [inline src/components/helpers/get-age-string.test.js:27]: Maybe add test with `null` and `undefined` if need it

File: package.json (hunk 1 above)
Comment (Vladosik12333) [reply package.json]: It's a coincidence.

File: src/components/helpers/get-age-string.test.js (hunk 2 above)
Comment (Vladosik12333) [reply src/components/helpers/get-age-string.test.js:27]: It is my bad. I stared fix, but forgot.
I agree. It it need.

**Review by akuzmichov-amplify — DISMISSED** [review]

File: src/components/helpers/get-age-string.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import getAge from "./get-age-string";
```
Comment (VadymKharyton) [inline src/components/helpers/get-age-string.test.js:1]: I see, that when running these tests locally, it says some Deprecation warning message about usage of moment package.
I know, that it's not related to your test changes, but it related to file, which you try to test. Please fix this warning and I will approve

**Review by VadymKharyton — CHANGES_REQUESTED** [review]

File: src/components/helpers/get-age-string.test.js (hunk 3 above)
Comment (Vladosik12333) [reply src/components/helpers/get-age-string.test.js:1]: fixed.

**Review by VadymKharyton — APPROVED** [review]
